### ISSUE TITLE
Corrupt Word File - Xml Parsing error Location: Part: /word/styles.xml

### ISSUE THREAD

**Issue by BSDGuyShawn** [sujith3g/docxtemplater-link-module#18]
The resulting Word document opens with a warning about an XML parsing error.

During recovery the repair simply lists Styles 1

![image](https://user-images.githubusercontent.com/1402771/32351680-78c6141e-bfdb-11e7-8db8-14b7d2d3844f.png)

![image](https://user-images.githubusercontent.com/1402771/32351685-81801f00-bfdb-11e7-98b6-5ff6be1572d7.png)

![image](https://user-images.githubusercontent.com/1402771/32351690-85f74662-bfdb-11e7-9094-c801c2da6f26.png)

I have been trying to submit a pull request to fix the issue but am unable to for some reason.

**Comment by BSDGuyShawn** [comment]
Looking at the repository I see a commit that addresses this issue (https://github.com/sujith3g/docxtemplater-link-module/commit/a2e9169096804c5afc555a1aab2494e0f82801c3#diff-bc7bf41a89317846b41ec59a65ca70b4), but it is not reflected on my system when I install the module with yarn or npm:
```
yarn add docxtemplater-link-module
yarn add v1.2.1
warning package.json: No license field
warning No license field
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
warning Your current version of Yarn is out of date. The latest version is "1.3.2" while you're on "1.2.1".
success Saved 1 new dependency.
└─ docxtemplater-link-module@0.2.3
warning No license field
Done in 1.22s.
```

**Comment by sujith3g** [comment]
released v0.2.4
